src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -454,11 +454,11 @@ private function resolveServices($value)
             $value = array_map(array($this, 'resolveServices'), $value);
         } elseif (is_string($value) &&  0 === strpos($value, '@=')) {
```

Comment:
as you extract the check for strings starting with ``@`` below and then handle the second char, you should include this case there as well